access/gitlab/app.go

```diff
@@ -7,6 +7,7 @@ import (
 
 	"github.com/gravitational/teleport-plugins/access"
 	"github.com/gravitational/teleport-plugins/utils"
+	"google.golang.org/grpc"
```

Comment:
If the default options are fine — then the client application doesn't have to import grpc.